test/integration/targets/canonical-pep517-self-packaging/runme.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+
+if [[ "${ANSIBLE_DEBUG}" == true ]]  # `ansible-test` invoked with `--debug`
+then
+    CLI_VERBOSE_FLAG=-v
+    SET_DEBUG_MODE=-x
+else
+    ANSIBLE_DEBUG=false
+    CLI_VERBOSE_FLAG=
+    SET_DEBUG_MODE=+x
+fi
+
+
+set -eEuo pipefail
+
+source virtualenv.sh
+
+set "${SET_DEBUG_MODE}"
+
+
+CLEANED_UP=false
+
+DIST_NAME=ansible_core
+
+LOWEST_SUPPORTED_BUILD_DEPS_FILE="${PWD}"/minimum-build-constraints.txt
+
+SRC_ROOT_DIR="$(dirname "$(dirname "$(dirname "$(dirname "${OUTPUT_DIR}")")")")"
+GENERATED_MANPAGES_SUBDIR=docs/man/man1/
+
+function git-clean-manpages() {
+    >&2 echo
+    >&2 echo '=>' Cleaning up the gitignored manpages...
+    # git \
+    #   --git-dir="${SRC_ROOT_DIR}"/.git \
+    #   --work-tree="${SRC_ROOT_DIR}" \
+    #   clean -dfX -- "${GENERATED_MANPAGES_SUBDIR}"
+    rm -rf ${CLI_VERBOSE_FLAG} "${GENERATED_MANPAGES_SUBDIR}"
+    >&2 echo '=>' Removed the generated manpages...
+    >&2 echo
+}
+
+
+TMP_DIR_SDIST_WITHOUT_MANPAGES=$(mktemp -d)
+TMP_DIR_SDIST_WITH_MANPAGES=$(mktemp -d)
+TMP_DIR_REBUILT_SDIST=$(mktemp -d)
+TMP_DIR_REBUILT_WHEEL=$(mktemp -d)
```

Comment:
Instead of using `mktemp`, why not use subdirectories under `$OUTPUT_DIR`? It's automatically cleaned up by `ansible-test` before each test run. Using it would simplify the test setup.

Reply:
Ah, I keep forgetting it's a thing. Makes sense.